x/harvest/keeper/borrow_test.go

```diff
@@ -27,8 +29,11 @@ func (suite *KeeperTestSuite) TestBorrow() {
 		loanToValueKAVA           sdk.Dec
 		priceBTCB                 sdk.Dec
 		loanToValueBTCB           sdk.Dec
+		priceBNB                  sdk.Dec
```

Comment:
These args are pretty verbose, could perhaps be simplified with a struct. Not a blocker though imo.

Reply:
Yeah, I'm planning on touching up the test config once we've got these initial tasks through to acceptance.